docs/questions/sharing/public-links.md

```diff
@@ -49,6 +47,23 @@ To create a public link that people can use to download the results of a questio
 
 ![Public export](../images/public-export.png)
 
+## Custom destinations in public links 
+
+If you want to create a drill-down pathway on a public question or dashboard, you can set up a [custom destination](../../dashboards/interactive.md) on the parent dashboard that goes to the public link of a child question or dashboard.
```

Comment:
Custom destinations aren't exclusive to simulating drill-down. It's just one thing you can do with custom destinations.

And it's only available for dashboards, so:

> If you want to create a drill-down pathway on a public question or dashboard

Is technically incorrect.

Maybe this section should have three subsections:

- Adding a custom destination to a public dashboard
- Simulating drill-down with custom destinations
- Passing filter values to a destination on click

or something like that.